ironfish-cli/src/commands/wallet/index.ts

```diff
@@ -36,11 +36,14 @@ export class AccountsCommand extends IronfishCommand {
       return []
     }
 
+    const defaultAccount = await client.wallet.getDefaultAccount()
```

Comment:
I think the extra request is OK, but we could avoid it if we add some `default` indicator to the response of `getAccountsStatus`

Reply:
Yea I thought of that, but it felt a little dirty to added a default flag to all account responses and have all of the `default=false` and just the one to `default=true`